.changeset/ripe-taxes-press.md

```diff
@@ -0,0 +1,16 @@
+---
+---
+---
+"section": other
+---
+
+Target ES2021
+
+The TypeScript build has been updated to target ES2021 instead of ES2020.
+This may result in newer JavaScript language features being used.
+This does not include updating the library version being used.
```

Comment:
The differences here may not be obvious. Any ideas to clarify? Maybe calling out that this just changes the JS that TypeScript outputs for our packages, while lib is the available APIs you can use when writing TS source? Ugh that might be more confusing.

Reply:
```suggestion
This does not change TypeScript types, nor the JavaScript APIs being used.
```